notebooks/variation_normalizer_rest_dp.py

```diff
@@ -1,5 +1,7 @@
 import requests
 
+""" TODO convert this to a jupyter notebook """
+
```

Comment:
We can remove the todo and add this to #242 

```suggestion
```

Reply:
yes